Formula/yarn.rb

```diff
@@ -17,7 +15,7 @@ class Yarn < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    cp_r '.', prefix
```

Comment:
BTW, it's weird that `bin.install_symlink Dir["#{libexec}/bin/*"]` is working 😕  It shouldn't.